modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/routes/ProjectRoutes.scala

```diff
@@ -46,7 +47,7 @@ final class ProjectRoutes(projects: Projects[Future])(implicit iamClient: IamCli
     idResolvable(p.id.value)
   }
 
-  private implicit val encoders: RoutesEncoder[Project] = new RoutesEncoder[Project](config.prefixes.coreContext)
+  private implicit val encoders: RoutesEncoder[Project] = new RoutesEncoder[Project]()
```

Comment:
RoutesEncoder can be a trait and then you can mixin with the `ProjectRoutes` class and the implicits will be picked directly (explained on another comment)